Packs/CortexXDR/Playbooks/Cortex_XDR_-_Port_Scan_README.md

```diff
@@ -61,4 +61,4 @@ There are no outputs for this playbook.
 
 ## Playbook Image
 ---
-![Cortex XDR - Port Scan](../doc_files/Cortex_XDR_-_Port_Scan.png)
\ No newline at end of file
+![Cortex XDR - Port Scan](https://raw.githubusercontent.com/demisto/content/0d732cbc4d9a4c06cb5266cca4acf5747e1663c0/Packs/CortexXDR/doc_files/Cortex_XDR_-_Port_Scan.png)
```

Comment:
Please use relative path like it was before.
If you replaced the PNG with the new image and kept the same file name it should work fine